components/DropdownLight/DropdownLight.jsx

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { colors, spacing, baseFontSize } from '../shared/theme';
+
+const itemPropType = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.shape({
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    label: PropTypes.string,
+  }),
+]);
+
+const DropList = styled.select`
+  width: 100%;
+  height: 44px;
+  font-size: ${baseFontSize};
+`;
+const OptionsItems = styled.option`
+  font-size: ${baseFontSize};
+`;
+
+const DropdownLight = ({
+  disabled,
+  label,
+  placeholder,
+  items,
+  selectedItem,
+  onChange,
+  theme,
+}) => (
+  <>
+    <DropList
```

Comment:
Why did you decide to use this name `DropList`?

Reply:
I changed it to DropDownSelect

Reply:
I changed it to DropDownSelect